webpack/index.js

```diff
@@ -1,11 +1,12 @@
 import componentRegistry from 'foremanReact/components/componentRegistry';
-import injectReducer from 'foremanReact/redux/reducers/registerReducer';
-import ForemanTemplates from './ForemanTemplates';
-import templateSyncReducer from './reducer';
+import React from 'react';
+import TemplatesRouter from './Router/TemplatesRouter';
+
+const ForemanTemplates = () => <TemplatesRouter />;
 
 componentRegistry.register({
   name: 'ForemanTemplates',
   type: ForemanTemplates,
 });
```

Comment:
Can this and `ForemanTemplates` should be converted to use `registerRoutes`? https://github.com/theforeman/foreman/blob/develop/developer_docs/client-routing.asciidoc